### PR TITLE
default images, warning dialogs

### LINKED ISSUE #32: Display better errors messages when an error happens

### PR DESCRIPTION
**Pull request by csell5**
adds warning and error modals to the favorites and sessions.

fixes default images on member pages.

closes #32

### CODE REVIEW

File: src/routes/my/Favorites.svelte [hunk 1]
```diff
@@ -34,9 +39,24 @@
...
+            text="Hold on a minute! You haven't favorited any ssssions yet. Get
+            on it!"
+          />
+        </div>
+      {/if}
     {:catch error}
-      <p>OH NO</p>
+      <ModalError
+        title="KABOOM!"
+        text="I'm sorry bugs are features right? If I were you, I'd refresh the
```
Comment (GeekOnCoffee) [inline src/routes/my/Favorites.svelte:56]: comma or semicolon after sorry

File: src/routes/my/Submissions.svelte [hunk 2]
```diff
@@ -36,9 +41,23 @@
...
+            title="You have no session submissions?"
+            text="Why not? Submit something today!"
+          />
+        </div>
+      {/if}
     {:catch error}
-      <p>OH NO</p>
+      <ModalError
+        title="KABOOM!"
+        text="I'm sorry bugs are features right? If I were you, I'd refresh the
```
Comment (GeekOnCoffee) [inline src/routes/my/Submissions.svelte:57]: Comma or semicolon after sorry

**Comment by GeekOnCoffee** [comment]
No favorites looks good; I already submitted a session, so I had to create a new account (created https://github.com/ThatConference/that.us/issues/140 to ease testing that) 

Looks like there's a z-index problem with the no submissions yet. 
Screen Shot 2020-07-21 at 8 19 24 PM

**Comment by csell5** [comment]
I created a new issue for that case where we need to make a better modal, especially when peoples profiles aren't logged in. ref #141 

Let's merge this for now.

**Pull request merged**